tests/lava/lib/peripherals/realsense/test_realsense.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import sys
+from pathlib import Path
+
+from lava.lib.peripherals.realsense.realsense import Realsense
+from lava.magma.core.run_configs import Loihi2SimCfg
+from lava.magma.core.run_conditions import RunSteps
+
+has_valid_camera = False
```

Comment:
`valid` seems like the wrong word here. How about `is_camera_available`?

Reply:
I changed this to `USE_CAMERA_TESTS` for consistency with the Prophesee test.